cfpb/consumer-credit-trends
# Lint js files

Cleaning up JS files by using our .eslintrc settings

## Changes

- Reformatted files based on linter errors
- Added JSDocs to undocumented functions
-

## Testing

- check out branch
- run `./setup-sh`
- run `gulp watch` and check page and charts for broken JS or JS errors (2 persistent errors from Github API are expected and can be ignored, these happen if your Personal Access Token has not been set up, see readme)
- run `gulp release` and check compiled files in `/charts/` for errors
- run 'gulp lint' to check for linter errors

## Review

- @marteki 
- @mistergone 

[Preview this PR without the whitespace changes](?w=0)

## Todos

I didn't fix 2 files with 'quote is redundant for object properties' errors.  These will be done in future refactors - when we have TDD set up so we can better catch any errors in changes to those important files
